Sources/FioriSwiftUICore/Views/ListPickerItem+View.swift

```diff
@@ -89,9 +89,29 @@ public struct ListPickerItemConfiguration {
             }
         }.typeErased
     }
+    
+    /// Creates a configuration object from a collection of data (conforms to `Identifiable`) which supports both signle-level and multi-level picker with the ability to select multiple items.
+    /// - Parameters:
+    ///   - data: The data for constructing the list picker.
+    ///   - children: The key path to the optional property of a data element whose value indicates the children of that element.
+    ///   - selection: A binding to a set which stores the selected items.
+    ///   - rowContent: The view builder which returns the content of each row in the list picker.
+    public init<Data, ID, RowContent>(_ data: Data,
+                                      children: KeyPath<Data.Element, Data?>?,
+                                      selection: Binding<Set<ID>>?,
+                                      @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+        where Data: RandomAccessCollection, RowContent: View, Data.Element: Identifiable, ID == Data.Element.ID
+    {
+        let id = \Data.Element.id
+        self.init(data, id: id, children: children, selection: selection, rowContent: rowContent)
+    }
 }
 
 public extension ListPickerItemConfiguration {
+    /// Creates a configuration object from a collection of `String` which supports both signle-level and multi-level picker with the ability to select multiple items.
```

Comment:
Spelling mistake `signle`

```suggestion
    /// Creates a configuration object from a collection of `String` which supports both single-level and multi-level picker with the ability to select multiple items.
```